### DISCUSSION TITLE
Embedded Node Tor Support

### DISCUSSION THREAD

**Discussion by shadowcpy** [ZeusLN/zeus#1941] [Q&A]
Hey everyone, is it possible to connect the embedded LND via Tor? I tried to use orbit, but LND reports "unknown ip type" when presented with tor nodes. Is that feature planned / already available?

**Comment by Darth-Coin** [comment]
For a mobile node Tor will only add more trouble than privacy. It's a false impression that Tor is offering privacy and for a mobile LN node will only add more latency for syncing and your node will be in real problem without syncing.
Read more about Zeus privacy here: https://docs.zeusln.app/for-users/embedded-node/privacy

To add more, I wrote this post on Stacker news about Tor nodes: https://stacker.news/items/944885
For the particular case of the Zeus embedded node + Tor:
- that node is not meant to be a public routing node, but a private (unannounced) node, for private use
- using neutrino to sync, it offers enough privacy to no link any xpub to the server you are connected to sync. Also you can use your own bitcoin node to connect if you want (if even better, because you can connect directly and closer as distance so you will have better sync).
- LN using onion routing, it offers enough privacy for a private (unannounced) channel. Also Zeus is offering wrapped invoices, that can act like a LNproxy for your mobile node.

In conclusion, running a Zeus embedded node in full Tor mode, is kind of useless and you are only shooting yourself in your own foot. Tor is not offering more privacy, is just an illusion of privacy.
